05/app.js

```diff
@@ -4,42 +4,51 @@ import { createRoot } from "react-dom/client";
 const root = createRoot(document.querySelector("#root"));
 
 class WeatherCheck extends React.Component {
-  constructor(props) {
-    super(props);
-    const {lat} = this.props;
-    this.state = {
-      data: `https://api.weatherbit.io/v2.0/current?key=44cf234f09214818a8e48efc2c273579&lat=${(lat="123")}&lon=-78.6382&lang=pl`,
-    };
-  }
+  state = {
+    data: null,
+  };
 
   render() {
     const { data } = this.state;
+    //console.log(data)
     if (data) {
+      const city = data[0].city_name;
+      const weather = data[0].weather.description;
+
       // renderuj dopiero jak pobierzesz dane z API
-      return <h1>informacje o pogodzie...</h1>;
+
+      return (
+        <section>
+          <h1>informacje o pogodzie... </h1>
+          <h2>Miasto: {city}</h2>
+          <h2>Pogoda: {weather}</h2>
+        </section>
+      );
     }
+    return null;
 
     // nic nie renderuj
-    return null;
+    //return null;
   }
 
   componentDidMount() {
-    const { data } = this.state;
-    const promise = fetch(data);
-
-    promise
+    const { lat, lot } = this.props;
+    fetch(
+      `https://api.weatherbit.io/v2.0/current?key=2a79607cb0e94d4590a92c16e676c48e&lat=${lat}&lon=${lot}&lang=pl`
+    )
```

Comment:
Teraz spróbuj przenieść zapytania do API do osobnego pliku - to zadanie dodatkowe ;)
Możesz się posiłkować tym artykułem: https://devmentor.pl/b/crud-w-osobnym-pliku-lepszy-kod-dla-api